src/Interpreters/InterpreterAlterQuery.cpp

```
@@ -282,6 +286,49 @@ BlockIO InterpreterAlterQuery::executeToDatabase(const ASTAlterQuery & alter)
             {
                 if (command.type == AlterCommand::MODIFY_DATABASE_SETTING)
                     database->applySettingsChanges(command.settings_changes, getContext());
+                else if (command.type == AlterCommand::MODIFY_DATABASE_COMMENT) {
+                    if (command.comment)
```

When will the `false` case occur? When no comment is provided? 

Should we throw an exception?